package.json

```diff
@@ -32,7 +32,7 @@
     "build-icons-prod": "node stack/tasks/icons.js PROD",
     "build-dev": "node stack/tasks/cleanup.js DEV && npm run build-css-dev && npm run build-html-dev && npm run build-js-dev && npm run build-icons-dev",
     "build": "node stack/tasks/cleanup.js PROD && node stack/tasks/build.html.js PROD && node stack/tasks/build-examples.js PROD && npm run build-css-prod && npm run build-js-prod && npm run build-lib && npm run build-icons-prod && node stack/tasks/copy-dist-to-docs.js",
-    "build-lib": "babel src --out-dir lib --ignore src/demos",
+    "build-lib": "babel src --out-dir lib --ignore src/demos --plugins transform-runtime",
```

Comment:
@LucaMele 
@pavestru 
For rollup we used the [`runtimeHelpers` option of rollup-plugin-babel](https://github.com/rollup/rollup-plugin-babel#helpers)
But we should really clean this up and have all within config files, not custom config overwrites in JS

https://github.com/axa-ch/patterns-library/search?q=runtimeHelpers&unscoped_q=runtimeHelpers

Reply:
generally, the whole build need to be cleanup up. it was a quick solution for a limited scope. :)

Reply:
Yeah, it really takes more time to craft a stable build and document all of it - not meant as accusation

Reply:
totally agree :)